value.go

```diff
@@ -361,6 +361,11 @@ func (v *Value) do(method string, params Params, body []byte) (value *Value, err
 	body, berr := ioutil.ReadAll(resp.Body)
 	resp.Body.Close()
 
+	// Go used to return an error if the Location header was missing before
+	// Go 1.19. For backwards-compatibility, we keep the same message.
+	if method == "GET" && resp.StatusCode == 303 && resp.Header.Get("Location") == "" {
+		return nil, errors.New("Get : 303 response missing Location header")
+	}
```

Comment:
@niemeyer Yes, I eventually realized I had been testing with a literal `%d` :-)

But I decided to change the approach as well for something more targeted and easier to explain.